scratch/camera_feed/display_camera_feed.cpp

```diff
@@ -0,0 +1,165 @@
+#include "../../src/network/network.hpp"
+#include "display_camera_feed.hpp"
+#include <stdlib.h>
+#include <string.h>
+#include <iostream>
+#include <turbojpeg.h>
+#include "SDL2/SDL.h"
+//#include <inttypes.h> to print uints
+
+#define CAMERA_MESSAGE_FRAME_DATA_MAX_SIZE 65000
+#define CAMERA_FRAME_BUFFER_SIZE 6220800
+#define CAMERA_FRAME_BUFFER_COUNT 5
+#define WINDOW_X_LENGTH 1920 //window resolution. Standard is 1920
+#define WINDOW_Y_LENGTH 1080 //my window resolution. Standard is 1080
+
+const long unsigned int _jpegSize = WINDOW_X_LENGTH * WINDOW_Y_LENGTH * 3;
+
+void decompress(unsigned char* buffer, unsigned char* compressed_image, int pitch){
+    tjhandle _jpegDecompressor = tjInitDecompress();
+    if(_jpegDecompressor == NULL)
+        std::cout << "decompressor was not made" << std::endl;
+    /*
+        The parameters are the decompressor, the compressed image, the size of the
+        image as an unsigned long, the buffer, the width of the jpeg, the pitch,
+        the height, the pixel type (We have RGB here each contained in 1 byte),
+        and the flags. Basically we say decompress it quickly.
+    */
+    //std::cout << ". JPG size: " << _jpegSize <<  ". Pitch " << pitch << std::endl;
+    if (tjDecompress2(_jpegDecompressor, compressed_image, _jpegSize, buffer, WINDOW_X_LENGTH, pitch, WINDOW_Y_LENGTH, TJPF_RGB, TJFLAG_FASTDCT) == -1)
+        std::cout << "decompressed incorrectly" << std::endl;
+    //free the decompressor.
+    tjDestroy(_jpegDecompressor);
+}
+
+int main(){
+    int pitch = 1920*3;
```

Comment:
This variable is defined here, but it is not used until line 136. I know what that variable is for since I know how the jpegturbo library works, but someone reading this code for the first time would assume that `pitch` and `offset` are very important variables since they are the first things defined at the top of main. Since they are only used for like 2 calls, define them right before they are used.

Also, when you move it, you do not need to give it an initial value. SDL will actually set the pitch for you.